src/modules/students.js

```diff
@@ -1,28 +1,28 @@
-const students = [
-  {
-    name: 'Ibrahim',
-    score: Math.floor(Math.random() * 100),
-  },
-  {
-    name: 'Abeer',
-    score: Math.floor(Math.random() * 100),
-  },
-  {
-    name: 'Tonia',
-    score: Math.floor(Math.random() * 100),
-  },
-  {
-    name: 'Omar',
-    score: Math.floor(Math.random() * 100),
-  },
-  {
-    name: 'Aml',
-    score: Math.floor(Math.random() * 100),
-  },
-  {
-    name: 'Reham',
-    score: Math.floor(Math.random() * 100),
-  },
-];
+// const students = [
```

Comment:
Great job @abdozayan12 🤝

- [ ] I would recommend you remove this file from your project since it is not used.